src/main/sensors/battery.c

```diff
@@ -434,6 +434,14 @@ uint8_t getBatteryCellCount(void)
     return batteryCellCount;
 }
 
+/**
+ * Gets average battery cell voltage in 0.01V units.
+ */
+uint16_t getBatteryAverageCellVoltage(void)
```

Comment:
Not sure if using a scaling for this that is different to the scaling of everything else is the right thing to do. Things like `osdGetBatterySymbol()` would become much easier to read if consistent scaling was used.